src/indexes.rs

```diff
@@ -726,4 +800,34 @@ mod tests {
 
         assert_eq!(status.len(), 0);
     }
+
+    #[async_test]
+    async fn test_get_one_update() {
+        let client = Client::new("http://localhost:7700", "masterKey");
+        let uid = "test_get_one_update";
+
+        let index = client.get_or_create(uid).await.unwrap();
+
+        let progress = index
+        .add_or_replace(
+            &[Movie {
+                name: "test".to_string(),
+                description: "no desc".to_string(),
+            }],
+            Some("name"),
+        )
+        .await
+        .unwrap();
```

Comment:
```suggestion
        let progress = index.delete_all_documents().await.unwrap();
```
In this test, we just need to get a `Progress`. `Client.delete_all_documents()` is the most compact way to do this. We can also remove the `Movie` struct.